README.md

```diff
@@ -1,5 +1,37 @@
 # Handling images in PDF file using C#
-PDF has become one of the most popular and widely used file formats, here images play an important role in PDF documents, images can grab our attention easily. So, in this article, you will learn how to add, extract, remove, and replace images from PDF documents using C#.
+The [Syncfusion .NET PDF library](https://www.syncfusion.com/document-processing/pdf-framework/net) allows users to add and manipulate images in PDF files. PDF has become one of the most popular and widely used file formats, here images play an important role in PDF documents. This repository contains examples to add, extract, remove and replace images from PDF documents using C#.
```

Comment:
The PDF has become one of the most popular and widely used file formats, here images play an important role in PDF documents.

Reply:
Changed the description